index.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+
+class AssetMap {
+
+  constructor(options) {
```

Comment:
can we add an option for cache: false, which will read from the file every time?   We'll use this option in development where the mapping file is changing regularly